diff-hl.el

```diff
@@ -851,10 +856,14 @@ The value of this variable is a mode line template as in
 (defvar diff-hl--magit-unstaged-files nil)
 
 (defun diff-hl-magit-pre-refresh ()
-  (setq diff-hl--magit-unstaged-files (magit-unstaged-files t)))
+  (unless (and diff-hl-disable-on-remote
+	       (file-remote-p default-directory))
```

Comment:
It seems like you're using tabs for indentation here. Could you untabify all the added lines?